core/src/main/java/de/danoeh/antennapod/core/util/DateUtils.java

```diff
@@ -37,9 +37,12 @@ public static Date parse(final String input) {
         }
         String date = input.trim().replace('/', '-').replaceAll("( ){2,}+", " ");
 
+        // remove colon from timezone to avoid differences between Android and Java SimpleDateFormat
+        date = date.replaceAll("([+-]\\d\\d):(\\d\\d)$", "$1$2");
```

Comment:
I hope that this does not break anything... Probably needs detailed beta tests.

Reply:
Yes, that has to be tested very well.